### ISSUE TITLE
Possible error in code - misleading indention

### ISSUE THREAD

**Issue by michaelortmann** [florian-sander/gseen.mod#15]
The following code is misleadingly indented as if it were guarded by the ‘else’:

https://github.com/florian-sander/gseen.mod/blob/e97cdf81c09168d62127671a2b60c7455ea9573c/do_seen.c#L449-L453

The whole surrounding code block looks off.

Same here:

https://github.com/florian-sander/gseen.mod/blob/e97cdf81c09168d62127671a2b60c7455ea9573c/do_seen.c#L488-L492

**Comment by florian-sander** [comment]
I agree, that looks off. I'm afraid I don't remember the thoughts behind those code blocks anymore and can't be of much help.

On the other hand, it doesn't seem to have caused any noticeable issues over the last 19 years... ¯\_(ツ)_/¯
